src/pybind/mgr/prometheus/module.py

```diff
@@ -794,6 +796,14 @@ def _setup_static_metrics(self) -> Dict[str, Metric]:
                 path,
                 'DF {}'.format(state),
             )
+        for state in DF_CLUSTER_BY_CLASS:
+            path = 'cluster_{}'.format(state)
+            metrics[path] = Metric(
+                'gauge',
+                path,
+                'DF {}'.format(state),
+                ('device_class',)
+            )
```

Comment:
Taking into account that the 3 metrics share the name with the ones already defined by `DF_CLUSTER` what about putting them in the existing for loop?

```py
for state in DF_CLUSTER:
  path = 'cluster_{}'.format(state)
  metrics[path] = Metric(...)
  
  path = 'cluster_by_class_{}'.format(state)
  metrics[path] = Metric(...)
```

Reply:
Updated accordingly